### PR TITLE
Availability new

### PR DESCRIPTION
**Pull request by almorcrette**

### CODE REVIEW

File: spec/features/available_dates_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  #   space = Space.create(name: 'Strawberry Fields Cottage', price: '150', description: 'Beautiful')
+  #   visit '/spaces/availability'
+  #   select "Strawberry Fields Cottage", :from => "space"
+  #   fill_in "date", :with => "2022/05/28"
+  #   click_button 'Submit'
+
+  #   expect(page).to have_content "Availability for Strawberry Fields Cottage has been added for 2022/05/28"
+  # end
+
+  scenario 'The host can provde a period with start and end dates to create availability' do
```
Comment (monenavarro) [inline spec/features/available_dates_spec.rb:12]: typo here for provide

File: spec/availability_spec.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    end
+  end
+
+  describe '.create_period' do
+    it 'creates multiple availabilities for one period' do
+      space2 = Space.create(name: 'Strawberry Fields Cottage', price: '150', description: "Strawberry Fields Cottage is a")
+      availability2 = Availability.create_period(start_date: "2022-05-28", end_date: "2022-06-30", space_id: space2.id)
+
+      #Not sure how to get this test to pass, we need to check the dates above are in the database for space id
+
```
Comment (mcsuGH) [inline spec/availability_spec.rb:27]: Maybe need a method to find availabilities for a space, then expect .length of the use of that method to return you the number of availabilities and make it equal to the number of days between those dates

File: spec/availability_spec.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+require './lib/availability'
+require_relative './database_helper'
+
+
+describe Availability do
+
+  let(:space) { Space.create(name: 'Strawberry Fields Cottage', price: '150', description: "Strawberry Fields Cottage is a") }
```
Comment (mcsuGH) [inline spec/availability_spec.rb:7]: Maybe need to use a double but not sure if it will break any tests

**Review by mcsuGH — APPROVED** [review]
Overall very good, seems to work great

File: spec/availability_spec.rb (hunk 2 above)
Comment (JordanManu) [reply spec/availability_spec.rb:27]: That's a good idea

**Pull request merged**